Integrations/integration-Cisco-umbrella.yml

```diff
@@ -567,7 +570,9 @@ script:
     outputs:
     - contextPath: Domain.Name
       description: Domain name
-    - contextPath: Domain.Score
+    - contextPath: Domain.Related.Name
+      description: Related domain name
+    - contextPath: Domain.Related.Score
```

Comment:
We do: `umbrela-domain-search`, `umbrela-domain-categorization`, `umbrella-ip-malicious-domains`

Do you think we need to do this somewhere else as well?